tcpdump.c

```diff
@@ -796,7 +796,7 @@ droproot(const char *username, const char *chroot_dir)
 	} else
 		error("Couldn't find user '%.32s'", username);
 #ifdef HAVE_LIBCAP_NG
-	/* We don't need CAP_SETUID, CAP_SETGID and CAP_SYS_CHROOT any more. */
```

Comment:
It looks like `any more` is correct British English:
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/definition/english/any-more)
* [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/any-more)
* [Collins](https://www.collinsdictionary.com/dictionary/english/any-more)

Reply:
Yes, but this repository and its sibling (see https://github.com/the-tcpdump-group/libpcap/pull/1198) tend to be predominantly en-US.

Reply:
"Any more" isn't US English?

For example, Robert Allen Zimmerman, born in Duluth, Minnesota, is generally considered to be from the US, and, [from something he wrote](https://www.bobdylan.com/songs/knockin-heavens-door/):

> Mama, take this badge off of me
> I can’t use it anymore

Reply:
And if the issue is "any more" vs. "anymore", [the Merriam-Webster Dictionary, a US English dictionary, says](https://www.merriam-webster.com/dictionary/anymore):

> Although both *anymore* and *any more* are found in written use, in current writing *anymore* is the more common styling.

so both are valid, although "anymore" is more common currently.

Reply:
That's how I usually split this hair:
* Would you like any more sugar?
* I do not eat sugar anymore.

Reply:
determiner/adverb:
https://dictionary.cambridge.org/fr/grammaire/grammaire-britannique/any-more-or-anymore